frontend/pages/redeem/[uuid]/thank-you.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Stack, Typography } from '@mui/material';
+import Lottie from 'lottie-react';
+import { NextPage } from 'next';
+import orangeTick from '../../../assets/orangeTickLottie.json';
+import { lottieSx, stackSx } from '../../../styles/redeem/thankYou';
+
+const ThankYou: NextPage = () => {
+  return (
+    <Stack component="main" maxWidth="md" spacing={2} sx={stackSx}>
+      <Box sx={lottieSx}>
+        <Lottie autoPlay loop={false} animationData={orangeTick} />
```

Comment:
Change lottie to the one i sent.

Reply:
Previous:

https://user-images.githubusercontent.com/7360964/195970284-3431df07-09a6-453d-ae54-5953ce12fd79.mp4


Current:


https://user-images.githubusercontent.com/7360964/195970292-6c6ec9c9-580f-443b-9d92-d182622f979f.mp4